hbeni/fgcom-mumble
# Sim connect support

Add SimConnect support trough RadioGUI as a bridge (Fix #45)

The idea is that RadioGUI connects to the SimConnect port, reads data from there and then translates it to the common fgcom-mumble UDP protocol.
The SimConnect connection can be connected using the "RadioGUI" menu. This will instanciate two COMs and deactivate all input (because they are now slaved to SimConnect).

**Testing**
- Open RadioGUI
- Connect SimConnect from the menu
- Start your sim
- See if the simulator changes are reflected in RadioGUI and the constructed FGCom UDP message (see bottom text field in radioGUI)
  - Toggle frequencies of COM1 and COM2
  - toggle electricity to the COMs
  - Use your plane's PTT button
  - Roll/fly around and see if the location information changes (lat, lon, height)
  - Change your callsign
